docs/src/pages/guides/alerts.md

```diff
@@ -61,24 +49,20 @@ The `response.headers` and `response.body` can be queried further with object ac
 
 For example, to trigger alert when content-type is not json you may use
 
-```json
-  "alerts" : [
-    {
-      "query": "response.headers['content-type'] != \"application/json\"",
+```yml
+  alerts : [
+    - query: response.headers['content-type'] != \application/json\
```

Comment:
this still needs double quote around `application/json`

```suggestion
    - query: response.headers['content-type'] != "application/json"
```

Reply:
ooops...  stripped the wrong thing.